test/clj_sqs_extended/test_fixtures.clj

```diff
@@ -107,6 +113,16 @@
   [handler-chan queue-opts & body]
   `(wrap-handle-queue ~handler-chan
                       test-standard-queue-name
+                      true
+                      {}
+                      ~queue-opts
+                      (fn [] ~@body)))
+
+(defmacro with-handle-queue-queue-opts-standard-no-autostop
```

Comment:
these fixture function variations are getting out of hand. you're basically using using fn names as arguments by this point

Reply:
this can be for v0.2

Reply:
I'll add this as another point to the #13 to be improved later on then.